app/src/main/java/org/mozilla/focus/state/AppReducer.kt

```diff
@@ -186,6 +187,13 @@ private fun showEraseTabsCfrChanged(state: AppState, action: AppAction.ShowErase
     return state.copy(showEraseTabsCfr = action.value)
 }
 
+/**
+ * The state of search widget snack bar changed
```

Comment:
nit: `snackbar` in Android is a single word.

Reply:
Done

Reply:
Still seeing two separate words. But not a blocker.